lib/ABI/FunctionType/Layout.cpp

```diff
@@ -90,12 +91,12 @@ class ToRawConverter {
   distributeNonPositionBasedArguments(const ArgumentSet &Arguments,
                                       std::size_t SkippedCount = 0) const;
 
-  std::pair<DistributedArgument, size_t>
-  considerRegisters(size_t Size,
-                    size_t AllowedRegisterLimit,
-                    size_t OccupiedRegisterCount,
-                    RegisterSpan Registers,
-                    bool AllowPuttingPartOfAnArgumentOnStack) const;
+  std::pair<DistributedArguments, std::size_t>
+  distributeImpl(model::QualifiedType Type,
+                 RegisterSpan Registers,
+                 std::size_t OccupiedRegisterCount,
+                 std::size_t AllowedRegisterLimit,
+                 bool AllowPuttingPartOfAnArgumentOnStack) const;
```

Comment:
@aleclearmind, please review the rework:
https://github.com/revng/revng/pull/232/commits/627853d03f9716a08a703987a93ae3316d19001c

Reply:
The commit is `ABI: rework non-position based distribution`.

Reply:
Fix indentation in commit message

Reply:
Done.